.gitignore

```diff
@@ -8,6 +8,9 @@
 !.gitignore
 !.npmignore
 
+# NPM
+package-lock.json
+
```

Comment:
package-lock.json should not be included into gitignore

Reply:
Removed